azure-event-hubs/1.0/modules/ROOT/pages/azure-event-hubs-connector-studio-configuration.adoc

```diff
@@ -0,0 +1,253 @@
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of the HTTP Listener source in the app.
+. On the *General* tab, specify the connection information for the connector.
+. On the *TLS* tab, optionally specify the TLS information for the connector.
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+
+[[add-connector-operation]]
+== Add a Connector Operation to the Flow
+
+When you add a connector operation to your flow, you are specifying an action for that connector to perform.
+
+To add an operation for Azure Event Hubs Messaging Connector, follow these steps:
+
+. In the *Mule Palette* view, select *Azure Event Hubs* and then select the desired operation.
+. Drag the operation onto the Studio canvas to the right of the source.
+
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+
+When you configure a connector, configure a global element that all instances of that connector in the app can use. Configuring a global element requires you to provide the authentication that the connector requires to access the target Azure Event Hubs system.
+Azure Event Hubs Connector supports *Sas authentication* and *Active directory authentication*.
+
+To configure the global element for Azure Event Hubs Connector, follow these steps:
+
+. Select the operation in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (*+*) next to the *Connector configuration* field to access the global element configuration fields.
+. On the *General* tab, in *Connection*, select the authentication method to configure:
+
+* <<config-sas, Sas authentication>>
+* <<config-active, Active directory authentication>>
+
+[start=4]
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+
+[[config-sas]]
+=== Configure Sas Authentication
+
+Enter the following information on the *General* tab of the *Global Element Properties* screen to configure *Sas Authentication*:
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Namespace a| String |  Event Hubs namespace name. |  | x
+| Shared Access Key Name a| String |  Shared access key given by Azure. |  | x
+| Shared Access Key a| String |  Shared access key value given by Azure. |  | x
+| Event Hub Name a| String |  Event Hub name. |  | x
+| Share Connection a| Boolean |  Toggles the builder to use the same connection for producers or consumers. |  false | 
+| Checkpoint Store Type a| One of:
+
+* <<AzureBlobStorageSasAuthentication>>
+* <<AzureBlobStorageActiveDirectoryAuthentication>> | Selects Azure Blob Storage Connection Provider. |  |
```

Comment:
We try not to use "Connection Provider" in the non-core connectors because it is a techie term. I just use "connection': Selects the Azure Blob Storage connection."